packages/create-cloudflare/turbo.json

```diff
@@ -3,18 +3,20 @@
 	"extends": ["//"],
 	"tasks": {
 		"build": {
+			"passThroughEnv": ["CI"],
```

Comment:
Make sure the CI env is available, but not part of the cache key

Reply:
I wonder if this should be set in the root turbo.json?

Reply:
157bf0f21